.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -7,12 +7,10 @@ body:
     attributes:
       label: Initial Checklist
       options:
-        - label:
-            I understand this is a bug report and questions should be posted in
+        - label: I understand this is a bug report and questions should be posted in
```

Comment:
I thought that starting text on the `- label:` line is only allowed if it fits in a single line... was that assumption wrong?

Reply:
I think so, this seems to be valid yaml: https://yaml-multiline.info/